datafusion/tests/sql.rs

```diff
@@ -797,20 +797,30 @@ async fn csv_query_count() -> Result<()> {
     Ok(())
 }
 
-// FIXME uncomment this when exec is done
-// #[tokio::test]
-// async fn csv_query_window_with_empty_over() -> Result<()> {
-//     let mut ctx = ExecutionContext::new();
-//     register_aggregate_csv(&mut ctx)?;
-//     let sql = "SELECT count(c12) over () FROM aggregate_test_100";
-//     // FIXME: so far the WindowAggExec is not implemented
-//     // and the current behavior is to throw not implemented exception
-
-//     let result = execute(&mut ctx, sql).await;
-//     let expected: Vec<Vec<String>> = vec![];
-//     assert_eq!(result, expected);
-//     Ok(())
-// }
+#[tokio::test]
+async fn csv_query_window_with_empty_over() -> Result<()> {
+    let mut ctx = ExecutionContext::new();
+    register_aggregate_csv(&mut ctx)?;
+    let sql = "select \
+    c2, \
+    row_number() over (),
+    sum(c3) over (), \
+    avg(c3) over (), \
+    count(c3) over (), \
+    max(c3) over (), \
+    min(c3) over () \
+    from aggregate_test_100 limit 5";
```

Comment:
I think an explicit order by is required (or sorting of results) to make this deterministic.

Reply:
Now the ordering is "just" a implementation detail of the batches getting merged / concatenated into 1 batch and keeping the same ordering.

Reply:
will add sort by c1